session/handler.go

```diff
@@ -338,7 +343,20 @@ func (h *Handler) adminListSessions(w http.ResponseWriter, r *http.Request, ps h
 		return
 	}
 
-	sess, total, nextPage, err := h.r.SessionPersister().ListSessions(r.Context(), active, opts, ExpandEverything)
+	var expandables Expandables
+	if urlValues.Has("expand") {
+		es := urlValues["expand"]
```

Comment:
You *could* reduce nesting here:

```suggestion
	if es, ok := urlValues["expand"]; ok {
```

Reply:
done